.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+defaults: &defaults
+  docker:
+    - image: circleci/node:8.10
+  working_directory: ~/circleci-deployment
+
+version: 2
+
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          key: yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - run: yarn
+      - save_cache:
+          paths:
+            - node_modules
+          key: yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+
+  test:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          key: yarn-cache-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - run: yarn test
+      - save_cache:
```

Comment:
Not sure if we need this at this point since build isn't run during this step?

Reply:
I left it because I wasn't sure if it would be needed when the cache was restored for deploy or acceptance -- you think it'll get it from `install`?

Reply:
Ah `yarn test` does do a build, I thought it did not. You should be able to keep this line and skip the `npm run build` commands in later jobs! But you can't do multiple keys under `restore_cache` as we found with a different repo. You have to repeat the command afaik.